langchain/vectorstores/chroma.py

```diff
@@ -228,7 +228,23 @@ def _similarity_search_with_relevance_scores(
         k: int = 4,
         **kwargs: Any,
     ) -> List[Tuple[Document, float]]:
-        return self.similarity_search_with_score(query, k)
+        """Return docs and relevance scores in the range [0, 1].
+
+        0 is dissimilar, 1 is most similar.
+
+        Args:
+            query: input text
+            k: Number of Documents to return. Defaults to 4.
+            **kwargs: kwargs to be passed to similarity search. Should include:
+                score_threshold: Optional, a floating point value between 0 to 1 to
+                    filter the resulting set of retrieved docs
+
+        Returns:
+            List of Tuples of (doc, similarity_score)
+        """
+
+        docs_and_scores = self.similarity_search_with_score(query, k)
+        return [(doc, 1.0 - score) for doc, score in docs_and_scores]
```

Comment:
i think chroma scores can also be squared l2 distance, which isn't normalized. we should figure out a way to get the distance for the collection and normalize based on that